### PR TITLE
Styling for inputs

### PR DESCRIPTION
**Pull request by ingedmundo**
As a developer I want a CSS class to user CrowdInt's inputs so that I can get the correct format
https://www.pivotaltracker.com/story/show/90807858

- Base style for inputs
- New javascript file (helper for the input behavior)

image

### CODE REVIEW

File: assets/stylesheets/colors.scss [hunk 1]
```diff
@@ -16,6 +16,9 @@ $silver:          #CCCCCC;
 $silver-dark:     #BBBBBB;
 $alto:            #DDDDDD;
```
Comment (mumoc) [inline assets/stylesheets/colors.scss]: maybe it's intentional but... alto? what does it means ?

File: branding.html [hunk 2]
```diff
@@ -130,6 +132,14 @@ <h3 class='text-center f-dust-gray'>Organization</h3>
     </section>
     <section class='main-fluid'>
       <section class='l-two-column'>
```
Comment (mumoc) [inline branding.html]: maybe it's not modified in this pr but you're having sectionitis xD

File: public/javascript/crowdint-styles.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+$(function() {
+  // This function checks if input fields has information captured.
+  // if so, it applies a class that changes the looknfeel of the input box.
+  $('input').keyup(function(){
```
Comment (efigarolam) [inline public/javascript/crowdint-styles.js]: Add a space between ) and {

File: assets/stylesheets/modules/inputs.scss [hunk 4]
```diff
@@ -0,0 +1,19 @@
+#{$all-text-inputs} {
+  @extend %lato-light;
+  background: $gallery;
+  border: 2px solid $gallery;
+  color: $mine-shaft;
+  font-size: 1rem;
+  max-height: 50px;
+  min-width: em(300);
+  padding: 1em;
+  &.has-info {
```
Comment (jparbros) [inline assets/stylesheets/modules/inputs.scss]: It is more readable if you add a line break

File: public/javascript/crowdint-styles.js (hunk 3 above)
Comment (mumoc) [inline public/javascript/crowdint-styles.js]: you could refactor this as:
```
$('input').keyup(function() { 
  hasInfo = $(this).val() != "";
  $(this).toggleClass('has-info', hasInfo);
});
```
also, consider adding a delay on the keyup ....but that's other story ^^

File: branding.html (hunk 2 above)
Comment (ingedmundo) [inline branding.html]: Yep I know... I will hve to change this dummie page, but not in this PR.

:P

File: public/javascript/crowdint-styles.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+    if( $(this).val() != ""){
+      $(this).addClass('has-info');
+    }else{
```
Comment (efigarolam) [inline public/javascript/crowdint-styles.js]: space before and after else

File: assets/stylesheets/colors.scss (hunk 1 above)
Comment (ingedmundo) [inline assets/stylesheets/colors.scss]: yep, it is, we are using a service for naming colors:

http://chir.ag/projects/name-that-color/#DDDDDD

File: public/javascript/crowdint-styles.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+$(function() {
+  // This function checks if input fields has information captured.
+  // if so, it applies a class that changes the looknfeel of the input box.
+  $('input').keyup(function(){
+    if( $(this).val() != ""){
```
Comment (efigarolam) [inline public/javascript/crowdint-styles.js]: space after )

File: public/javascript/crowdint-styles.js (hunk 3 above)
Comment (mumoc) [inline public/javascript/crowdint-styles.js]: In fact, I think you could just do it as:
```
$('input').keyup(function() { 
  hasInfo = $(this).val();
  $(this).toggleClass('has-info', hasInfo);
});
```

File: assets/stylesheets/modules/inputs.scss [hunk 7]
```diff
@@ -0,0 +1,20 @@
+#{$all-text-inputs} {
+  @extend %lato-light;
+  background: $gallery;
+  border: 2px solid $gallery;
+  color: $mine-shaft;
+  font-size: 1rem;
```
Comment (mumoc) [inline assets/stylesheets/modules/inputs.scss]: I like the use of the new unit (root-em) but, what about IE 7 u 8 and Opera ... <11 ... ?? I would suggest to have a fallback
```
font-size: 10px;
font-size: 1rem;
```
or something similar; you could also create a mixin or a class to define the size and provide compatibility ...

Comment (mumoc) [inline assets/stylesheets/modules/inputs.scss]: http://caniuse.com/#feat=rem

File: assets/stylesheets/colors.scss (hunk 1 above)
Comment (mumoc) [inline assets/stylesheets/colors.scss]: jeje great, the page I shared time ago, I found it useful; I guess we could standardize the use of it ... I mean, for naming colors

File: public/javascript/crowdint-styles.js [hunk 8]
```diff
@@ -0,0 +1,8 @@
+$(function() {
+  // This function checks if input fields has information captured.
+  // if so, it applies a class that changes the looknfeel of the input box.
+  $('input').keyup(function() {
```
Comment (ingilniero) [inline public/javascript/crowdint-styles.js]: I'm not sure about applying this to all the inputs in the page, maybe you could add a class for the inputs that you want to have this style, being explicit about it. What do you think @mumoc?

Comment (ingedmundo) [inline public/javascript/crowdint-styles.js]: This is supposed to be the default behavior for all inputs when using crowdint-styles, using a class will make you to specify ```input type='text' class='gil'```.

Comment (ingilniero) [inline public/javascript/crowdint-styles.js]: Oh! ok, makes sense.

File: assets/stylesheets/modules/inputs.scss (hunk 7 above)
Comment (hecbuma) [inline assets/stylesheets/modules/inputs.scss]: do we care about IE8 ? it's gonna be out of official support this year, if I'm not wrong

File: public/javascript/crowdint-styles.js (hunk 8 above)
Comment (hecbuma) [inline public/javascript/crowdint-styles.js]: I'll go with the class. what if you have an input and for some reason you don't want styles in there, it's gonna be a pain remove them for that specific case

**Pull request merged**